CAIPs/caip-261.md

```diff
@@ -0,0 +1,384 @@
+"type": ["VerifiableCredential", "PeerTrustCredential"],
+"issuanceDate": "2024-02-15T07:05:56.273Z",
+"issuer": "did:pkh:eip155:1:0x44dc4E3309B80eF7aBf41C7D0a68F0337a88F044",
+"credentialSubject":
+{
+  "id": "did:pkh:eip155:1:0xC3764761E297D6f121e79C32A65829Cd1dDb4D33",
+  "trustworthiness":
+  [
+    {
+      "scope": "Honesty",
+      "level": -1
+      "reason": ["Scam", "Rug pull"]
+    },
+    {
+      "scope": "Data protection",
+      "level": -1,
+      "reason": ["Data monetization", "Data leak"]
+    },
+    {
+      "scope": "Software security",
+      "level": -0.5,
+      "reason": ["Poor track record", "Lack of transparency"]
+    },
+    {
+      "scope": "User experience design",
+      "level": -0.5,
+      "reason": ["Poor UX"]
+    },
+    {
+      "scope": "Lawful",
+      "level": -1,
+      "reason": ["Money laundering", "Piracy", "Non-compliance"]
+    }
+  ]
+},
+"credentialSchema": [{
+  "id": "ipfs://QmcwYEnLysTyepjjtJw19oTDwuiopbCDbEcCuprCBiL7gl",
+  "type": "JsonSchema"
+},
+"proof": {}
+```
+*The example represents an assertion issued by the issuer identified as did:pkh:eip155:1:0x44dc4E3309B80eF7aBf41C7D0a68F0337a88F044. 
+It expresses distrust towards the subject identified as did:pkh:eip155:1:0xC3764761E297D6f121e79C32A65829Cd1dDb4D33 in areas such as honesty, software development, and software security to some extent. 
+The assertion provides reasons for each scope.*
+
+#### Validity Period
+Trust, being inherently dynamic, can be managed within the document by specifying a validity period, after which the credential must be renewed to maintain its validity.
+A validity limitation can be introduced with the fields `validFrom` and `validUntil`.
+```json
+"@context": ["https://www.w3.org/2018/credentials/v1"],
+"type": ["VerifiableCredential", "PeerTrustCredential"],
+"issuanceDate": "2024-02-15T07:05:56.273Z",
+"issuer": "did:pkh:eip155:1:0x44dc4E3309B80eF7aBf41C7D0a68F0337a88F044",
+"validFrom": "2024-01-01T19:23:24Z",
+"validUntil": "2025-01-01T19:23:24Z",
+```
+
+### Trust Management
+Initially, issued trust assertions should be properly persisted to ensure their availability according to the use-case.
+Subsequently, as trust between peers evolves over time, this proposal outlines a method for managing the entire lifecycle of these assertions, encompassing their creation, update, and eventual revocation.
+
+#### Trust Peristance
+Trust Assertions issued by peers are composing a verifiable trust graph that can be consumed for trustless reputation-based use cases. To fulfill its purpose, these assertions SHOULD be persisted using mechanisms that meet these properties relative to the use-case:
+- Security,
+- Data integrity,
+- Data ownership,
+- Interoperabiltiy,
+- Scalability,
+- Censorship resistance,
+- Sustainability.
+
+#### Trust Update
+When an issuer needs to update a trust assertion, they simply generate a new assertion containing the revised information. 

```

Comment:
is there any tombstoning system for assertions that have been superceded? could/should there be, at least optionally? people unfamiliar with IPFS may not realize they have manage their own garbage collection in a system where superceding/updating records are submitted often...